### DISCUSSION TITLE
My account was stolen, technical support has not responded for two weeks now, what should I do?

### DISCUSSION THREAD

**Discussion by panikajoreserv** [community/community#128973] [Other Feature Feedback, Questions, & Ideas]
In general, most likely I caught a virus, and the next day I discovered that my email address on Github had changed to another and my password had been changed. I immediately wrote to technical support but they did not answer me, what should I do?

**Comment by ebndev** [comment]
Hi @panikajoreserv,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
